python_modules/dagster/dagster/core/asset_defs/assets_job.py

```diff
@@ -179,3 +194,39 @@ def _op():
         return io_manager.load_input(input_context_with_upstream)
 
     return _root_manager
+
+
+def build_partitions_info(
```

Comment:
Maybe `build_job_partitions_from_assets` or something like that?  Need to distinguish between top-level computational partition info from the asset partition info.